src/variant/basis.cpp

```diff
@@ -488,7 +488,7 @@ Vector3 Basis::get_euler(EulerOrder p_order) const {
 				euler.z = 0.0f;
 			}
 			return euler;
-		} break;
+		}
```

Comment:
This is fine, but also needs to be corrected in the core engine that this code is synced with. Otherwise it's unnecessary diff for future comparisons.

Reply:
godotengine/godot#68861